src/AppInstallerRepositoryCore/CompositeSource.cpp

```diff
@@ -1054,14 +1099,17 @@ namespace AppInstaller::Repository
                 auto names = installedVersion->GetMultiProperty(PackageVersionMultiProperty::Name);
                 auto publishers = installedVersion->GetMultiProperty(PackageVersionMultiProperty::Publisher);
 
+                Utility::NameNormalizer normer(Utility::NormalizationVersion::Initial);

```

Comment:
Different versions of the index will use different versions of the normalization; I don't think that this is future proof.  The goal was that from the outside, normalization was known to be used, but not understood.  I don't think that this level of detail is a workable solution for an arbitrary source to implement.